app/assets/javascripts/teams.js

```diff
@@ -1,5 +1,5 @@
 jQuery(function ($) {
-  $('#add_team_user_btn').on('click', function (_event) {
+  $('#add_team_user_btn').on('click', function () {
```

Comment:
Is this parameter really optional then ? I wrote `_event` to state "jQuery wants to pass a parameter but we don't use it". If so, I think that we can strip most of the parameters that start with "_" 😄